src/App.tsx

```diff
@@ -42,6 +49,8 @@ export const App: React.FC = () => {
 
   activateIndents();
 
+  const clearDomainName: string = checkDomainName(domainName);
```

Comment:
Please, keep components not overhead with external functionality.
Move  `clearDomainName` to its module.